test/integration/cluster_solver/shared_tools/accumulation/tp/CMakeLists.txt

```diff
@@ -1,8 +1,7 @@
 dca_add_gtest(ringG_tp_accumulator_gpu_test
-        EXTENSIVE
-        GTEST_MAIN
-        CUDA
-        MPI MPI_NUMPROC 3
-        INCLUDE_DIRS ${DCA_INCLUDE_DIRS};${PROJECT_SOURCE_DIR}
-        LIBS     ${DCA_LIBS} parallel_mpi_concurrency
-        )
+ FAST
+ CUDA_MPI
+ MPI MPI_NUMPROC ${DCA_TEST_GPU_COUNT}
```

Comment:
Here we use the number of GPU per node. We definitely want to run the ringG test with all the GPU's on a node.

Reply:
Why? This is a unit test, not a performance test, and it should provide correct results for any number of processes  > 1.

Reply:
It does but running it with 1 process provides a false result as there is no actual testing of the mpi element of it.  So it should not run and be marked passed